src/Raven.Client/Documents/Session/DocumentQuery.cs

```diff
@@ -42,6 +43,13 @@ public IDocumentQuery<TProjection> SelectFields<TProjection>()
             });
         }
 
+
+        public IDocumentQuery<TTs> SelectTimeSeries<TTs>(Func<ITimeSeriesQueryBuilder, TTs> timeSeriesQuery)
```

Comment:
Let's change to explicit interface implementation. It is way easier to track code changes then when interface changes.

Reply:
done